ipmiMgrApp/Db/system_common.template

```diff
@@ -244,8 +244,8 @@ record(mbbo, "$(P)$(R)$(id)SensorScanPeriod-Sel") {
  field(FRVL, "4")
  field(FRST, "60 seconds")
  field(OUT,  "#B0 C0 N0 @$(link)+scan")
- field(VAL,  "1")
- field(RVAL, "1")
+ field(VAL,  "0")
+ field(RVAL, "0")
```

Comment:
Ok, I think we should only merge this change if we can confirm that adding enough delay between messages to avoid MCH errors still allows it to run in less than 5s.

Reply:
We found out that the MCH warnings is not being caused because there are a lot of requisitions nor the delay between this requisitions. The main hypothesis is that it is from some statistical misleading communication between MCH and MMC, so adding delay between messages don't fix the problem